.gitignore

```diff
@@ -0,0 +1 @@
+bazel-*
```

Comment:
I think it can remain in github because anyone cloning this repo need bazel cache files locally, which reduces fetching time, and it should should be ignored by git.

Reply:
Agree. I think listing files here is better that using glob pattern:

bazel-out
bazel-bin
bazel-testlogs